grunt_tasks/coffee_discovering_task.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function(grunt) {
```

Comment:
It's an open question : for now I moved only what was strongly related to open-orchestra. The remaining tasks are generics so I leave there where they are.
But another way to think is effectively to move all tasks to open-orchestra-cms-bundle. What is your preference ?